sqlite_query.c

```diff
@@ -142,20 +427,60 @@ sqlite_convert_to_pg(Oid pgtyp, int pgtypmod, sqlite3_stmt * stmt, int stmt_coli
 									valstr,
 									attinmeta->attioparams[attnum],
 									attinmeta->atttypmods[attnum]);
-	return (struct NullableDatum) { value_datum, false};
+	return (struct NullableDatum){value_datum, false};
+}
+
+int sqlite_bind_blob_algo (int attnum, Datum value, sqlite3_stmt * stmt)
```

Comment:
Please add comment for new function. And use style:
```c
/*
 * Function purpose
 */
int
sqlite_bind_blob_algo (int attnum, Datum value, sqlite3_stmt * stmt)
{

}
```

Reply:
Added.

Reply:
Thank for updating. Confirmed.

Reply:
Resolved.